### PR TITLE
SERVER_URL variable

### PR DESCRIPTION
**Pull request by vtols**

### CODE REVIEW

**Comment by rodionos** [comment]
Docs need to be updated describing what syntax it supported by SERVER_URL

**Comment by vtols** [comment]
Should header and footer be configurable from properties file too?

**Comment by axibase-svn** [comment]
yes, i forgot to add to the list of configurable options

File: README.md [hunk 1]
```diff
@@ -79,7 +145,7 @@ The list of possible user templates:
 - slack
 - telegram
 
-Each Webhook user will have the same name. Webhooks URLs are defined as described [here](https://github.com/axibase/atsd/blob/master/api/data/messages/webhook.md#sample-urls)
+Each webhook user will have the same name as template. Webhook URLs are defined as described [here](https://github.com/axibase/atsd/blob/master/api/data/messages/webhook.md#sample-urls)
```
Comment (rodionos) [inline README.md]: Provide an example of output that will be presented to the end user if he enabled AWS and github webhooks.

**Pull request merged**